### PR TITLE
Adding vuid cmd draw indexed indirect 02710

### PR DESCRIPTION
**Pull request by ziga-lunarg**
Check if offset in CmdDrawIndexedIndirect is a multiple of 4

### CODE REVIEW

**Comment by sfricke-samsung** [comment]
if looking for more low hanging fruit, just realized `VUID-vkCmdDrawIndirect-offset-02710` is also missing

File: layers/parameter_validation_utils.cpp [hunk 1]
```diff
@@ -3936,6 +3936,11 @@ bool StatelessValidation::manual_PreCallValidateCmdDrawIndexedIndirect(VkCommand
                          "CmdDrawIndexedIndirect(): drawCount (%u) is not less than or equal to the maximum allowed (%u).",
                          drawCount, device_limits.maxDrawIndirectCount);
     }
+    if (offset & 3) {
+        skip |= LogError(commandBuffer, "VUID-vkCmdDrawIndexedIndirect-offset-02710",
```
Comment (ncesario-lunarg) [inline layers/parameter_validation_utils.cpp:3940]: Looks like there are a few of these that might be worth adding in this PR (looks like there are 6 total?). One approach would be to add these to the appropriate [DrawDispatchVuid](https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/master/layers/drawdispatch.cpp#L46) structs and then use `GetDrawDispatchVuid` to get the appropriate struct for each API call. Or you can add them manually to each API call.

**Review by ncesario-lunarg — COMMENTED** [review]
LGTM, though would be nice to get the rest of the "02710 variants" in as well.

File: layers/parameter_validation_utils.cpp (hunk 1 above)
Comment (sfricke-samsung) [reply layers/parameter_validation_utils.cpp:3940]: So I guess the idea of `DispatchVuidsCmdDraw` was to handle the Common VU files

But I guess there is some benefit having them manually in `StatelessValidation` too

I guess no opinion on either approach, just that a decision should be made

Comment (ncesario-lunarg) [reply layers/parameter_validation_utils.cpp:3940]: Looks like using `DrawDispatchVuid`s would be a refactor project, so I'd say just add the VUIDs manually (should be mostly copy/paste?) to the corresponding `Cmd*Draw*` checks.

Comment (spencer-lunarg) [reply layers/parameter_validation_utils.cpp:4967]: So I looked into this, there is some ugliness with `vkCmdDrawIndirectCount`/`vkCmdDrawIndirectByteCountEXT`/`vkCmdDrawMeshTasksIndirectEXT` that might require a spec fix first before moving to `DrawDispatchVuids` (which I want to do)

**Review by spencer-lunarg — APPROVED** [review]

**Pull request merged**